sample-apps/optional-add-ons-cart-transform/extensions/optional-add-ons-rust/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "optional-add-ons-rust"
+version = "1.0.0"
+edition = "2021"
+rust-version = "1.62"
+
+[dependencies]
+serde = { version = "1.0.13", features = ["derive"] }
+serde_json = "1.0"
+shopify_function = "0.4.0"
```

Comment:
Might as well bump this to the latest 0.6.0 that was released last week. This will be evergreen longer.

Reply:
Good call!